pom.xml

```diff
@@ -112,12 +112,14 @@
 		<benchmarkjar.name>benchmarks</benchmarkjar.name>
 		<javac.target>1.8</javac.target>
 		<slf4j.version>1.7.21</slf4j.version>
-		<log4j2.version>2.6.2</log4j2.version>
+		<log4j2.version>2.1</log4j2.version>
```

Comment:
Maybe I'm confused, but isn't that what I'm doing?...  If you look at where I imported storm-core in the pom, I have a bunch of expulsions for logging...